src/items/implementations.md

```diff
@@ -1,10 +1,62 @@
 # Implementations
 
-An _implementation_ is an item that can implement a [trait](items/traits.html) for a
-specific type.
+An _implementation_ is an item that associates items with an *implementing type*.
+
+There are two types of implementations: Bare implementations and [trait] 
+implementations.
 
 Implementations are defined with the keyword `impl`.
 
+## Bare Implementations
+
+A bare implementation is defined as the sequence of the `impl` keyword, generic
+type declarations, a path to a nomial tyupe, a where clause, and a bracketed
+set of associable items.
+
+The nominal type is called the *implementing type* and the associable items are
+the *associated items* to the implementing type.
+
+Bare implementations associate the associated items to the implementing type.
+
+The associated item has a path of a path to the implementing type followed by
+the associate item's path component.
```

Comment:
Maybe note that a bare impl cannot have associated type( aliase)s.

Reply:
👍 I had told myself to do that a million times, yet I did not do it.

Reply:
(review note, this is resolved below)